docs/project/development-guide.md

```diff
@@ -1,72 +1,436 @@
+    - [(Contrib) Running tests for Postgres online store](#contrib-running-tests-for-postgres-online-store)
+    - [(Contrib) Running tests for HBase online store](#contrib-running-tests-for-hbase-online-store)
+- [(Experimental) Feast UI](#experimental-feast-ui)
+- [Feast Java Serving](#feast-java-serving)
+- [Developing the Feast Helm charts](#developing-the-feast-helm-charts)
+  - [Feast Java Feature Server Helm Chart](#feast-java-feature-server-helm-chart)
+  - [Feast Python / Go Feature Server Helm Chart](#feast-python--go-feature-server-helm-chart)
+- [Feast Go Client](#feast-go-client)
+  - [Environment Setup](#go-environment-setup)
+  - [Building](#building-go)
+  - [Code Style & Linting](#go-code-style--linting)
+  - [Unit Tests](#go-unit-tests)
+  - [Testing with GitHub Actions workflows](#testing-with-github-actions-workflows)
+- [Data Storage Formats](#feast-data-storage-format)
+## Overview
+This guide is targeted at developers looking to contribute to Feast components in
+the main Feast repository:
+- [Feast Python SDK / CLI](#feast-python-sdk--cli)
+- [Feast Java Serving](#feast-java-serving)
+- [Feast Go Client](#feast-go-client)
 
-* [Project Structure](development-guide.md#repository-structure)
-* [Making a Pull Request](development-guide.md#making-a-pull-request)
-* [Feast Data Storage Format](development-guide.md#feast-data-storage-format)
-* [Feast Protobuf API](development-guide.md#feast-protobuf-api)
-* [Maintainer Guide](./maintainers.md)
+Please see [this page](https://docs.feast.dev/reference/codebase-structure) for more details on the structure of the entire codebase.
 
-> Learn How the Feast [Contributing Process](contributing.md) works.
+## Compatibility
 
-## Making a Pull Request
+The Feast project 
 
-{% hint style="info" %}
-See also the CONTRIBUTING.md in the corresponding GitHub repository \(e.g. [main repo doc](https://github.com/feast-dev/feast/blob/master/CONTRIBUTING.md)\)
-{% endhint %}
+## Community
+See [Contribution process](https://docs.feast.dev/project/contributing) and [Community](https://docs.feast.dev/community) for details on how to get more involved in the community.
 
-### Incorporating upstream changes from master
+A quick few highlights:
+- [RFCs](https://drive.google.com/drive/u/0/folders/0AAe8j7ZK3sxSUk9PVA)
+- [Community Slack](https://slack.feast.dev/)
+- [Feast Dev Mailing List](https://groups.google.com/g/feast-dev)
+- [Community Calendar](https://calendar.google.com/calendar/u/0?cid=ZTFsZHVhdGM3MDU3YTJucTBwMzNqNW5rajBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+  - Includes biweekly community calls at 10AM PST
 
-Our preference is the use of `git rebase` instead of `git merge` : `git pull -r`
+## Making a pull request
+We use the convention that the assignee of a PR is the person with the next action.
 
-### Signing commits
+This means that often, the assignee may be empty (if no reviewer has been found yet), the reviewer, or the PR writer if there are comments to be addressed.
```

Comment:
sentence doesn't quite make sense to me

Reply:
Came in from the CONTRIBUTING file but I'll fix